### ISSUE TITLE
Make sidebar-menu-item component accessible for keyboard navigation

### ISSUE THREAD

**Issue by aniri** [code4romania/taskforce-fe-components#169]
It should be possible to navigate between sidebar menu items using the tab key.

Suggestions:
- add `tabindex` property - https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets#Using_tabindex
- user aria attributes - https://www.w3.org/TR/wai-aria-practices/

**Comment by aniri** [comment]
please use fix for https://github.com/code4romania/taskforce-fe-components/pull/174 as reference ;)

**Comment by aniri** [comment]
closing as it was fixed
